targets/xfce

```diff
@@ -9,19 +9,15 @@ CHROOTBIN='crouton-noroot startxfce4'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-# Ubuntu 12.04 has an old version of Xfce, but there's a PPA for Xfce 4.10.
+# Ubuntu 12.04 no longer has a PPA for Xfce 4.10 - so remove it if found.
 if release -eq precise && [ "$ARCH" = 'amd64' -o "$ARCH" = 'i386' ]; then
-    # Pull in the signing key
-    apt-key adv --keyserver 'keyserver.ubuntu.com' \
-                --recv-keys '07D44A5424C7C12A4BAB1E80EB563F93142986CE'
-    # Add the software source
-    cat > '/etc/apt/sources.list.d/xfce-4.10.list' <<EOF
-# Bring in Xfce 4.10 on Ubuntu 12.04
-deb http://ppa.launchpad.net/xubuntu-dev/xfce-4.10/ubuntu $RELEASE main
-deb-src http://ppa.launchpad.net/xubuntu-dev/xfce-4.10/ubuntu $RELEASE main
-EOF
-    # Update database
-    apt-get update || true
+    if [ -s /etc/apt/sources.list.d/xfce-4.10.list ]; then
+        rm /etc/apt/sources.list.d/xfce-4.10.list*
+        # Remove PPA key
+        apt-key del 142986CE
```

Comment:
I'd rather not remove the key in the event that the user added another xubuntu-dev ppa (such as the 4.12 one).